layers/+emacs/org/packages.el

```diff
@@ -621,6 +621,7 @@ Headline^^            Visit entry^^               Filter^^                    Da
 
 (defun org/init-org-jira ()
   (use-package org-jira
+    :after org
```

Comment:
This causes the `SPC a o J` key bindings not to load on startup.

And it won't be needed for the new key bindings org mode key bindings, because this:
`(spacemacs/set-leader-keys-for-major-mode 'org-mode`
probably means that the key bindings will be defined when `org-mode` loads.